app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -222,4 +235,37 @@ public void showAccountChooserDialog(CharSequence dialogTitle, boolean showActiv
             .setAdapter(adapter, (dialogInterface, index) -> listener.onAccountSelected(accounts.get(index)))
             .show();
     }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        if (requestCode != REQUESTER_NONE && requesters.containsKey(requestCode)) {
+            requesters.remove(requestCode).onRequestPermissionsResult(permissions, grantResults);
+        }
+    }
+
+    public boolean requestPermissions(String[] permissions, int minimumBuildVersion, @Nullable PermissionRequester requester) {
+        if (minimumBuildVersion == BUILD_VERSION_ANY || Build.VERSION.SDK_INT >= minimumBuildVersion) {
```

Comment:
Do we even need this check? `checkSelfPermission` should just return true?.

Reply:
I don't know? I assumed the original check was there for a reason. :smiley:

Reply:
Yeah, maybe leave it there